src/Core/AdminConsole/Repositories/IOrganizationUserRepository.cs

```diff
@@ -58,4 +58,6 @@ UpdateEncryptedDataForKeyRotation UpdateForKeyRotation(Guid userId,
     /// Returns a list of OrganizationUsers with email domains that match one of the Organization's claimed domains.
     /// </summary>
     Task<ICollection<OrganizationUser>> GetManyByOrganizationWithClaimedDomainsAsync(Guid organizationId);
+
+    Task SetOrganizationUsersStatusAsync(IEnumerable<Guid> userIds, OrganizationUserStatusType status);
```

Comment:
Not all `OrganizationUserStatusType` changes are so simple. Changing from Invited -> Accepted and Accepted -> Confirmed requires other updates to the data (e.g. linking the Organization User to the User, doing key exchanges), so this cannot be used for those transitions. At the moment it would be limited to revoking users and restoring users. Which is fine, but how can we limit this interface so it's not misused? Maybe the SQL is general purpose, but the repository interface remains specific to the revoke action?